clients/src/main/java/org/apache/kafka/common/security/authenticator/LoginManager.java

```diff
@@ -97,7 +97,7 @@ private LoginManager(JaasContext jaasContext, String saslMechanism, Map<String,
      */
     public static LoginManager acquireLoginManager(JaasContext jaasContext, String saslMechanism,
                                                    Class<? extends Login> defaultLoginClass,
-                                                   Map<String, ?> configs) throws LoginException {
+                                                   Map<String, ?> configs) throws Exception {
```

Comment:
As above, we don't need the signature change if the above is not changed?

Reply:
Sorry about the unintentional mistake - I've reverted these changes.